src/static/images/intro-background-fit.svg

```diff
@@ -0,0 +1,13 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="1920" height="1250">
+  <defs>
+    <style>
+      .cls-1{fill:#a8caba;fill-rule:evenodd;opacity:.05}
```

Comment:
Curious why this is styled in CSS as opposed to attributes like the other elements? Can we style using elements for consistency?

Reply:
If I'm not mistaken, this is an optimization made by the editing tools like Adobe Illustrator, or by SVGO due to the face that in this case `.cls-1` is used by three SVG `path`s and in terms of bytes, it is more efficient to have a CSS rule defined once and then apply it thrice. I didn't do the no of chars/bytes math though.

Either way, looks clean and DRY to me :)